procrastinate/aiopg_connector.py

```diff
@@ -87,8 +87,11 @@ def __init__(
     ):
         """
         Create a PostgreSQL connector using aiopg. The connector uses an ``aiopg.Pool``,
-        which is either created automatically upon first use, or set through the
-        `AiopgConnector.set_pool` method.
+        which is either created by `App.open_async`.
+
+        The pool connection parameters can be provided in this method. Alternatively,
```

Comment:
It's the constructor, I wouldn't say "in this method" here.

Reply:
I changed for `The pool connection parameters can be provided here.`